### ISSUE TITLE
Hoobs 4.0 with Hue

### ISSUE THREAD

**Issue by cdaniels77** [hoobs-org/HOOBS#1402]
**Description**
Lights are not showing up in Accessories/ Homekit

**Expected behavoir**
Lights to show up in accessories/ homekit

**Plugin name**
Hue Philips

**Please report the issue to the plugin developer**
We like to help the community, however some issues are out of our control. Please report this to the plugin developer and include a link here.

**Version**
4.0.92

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [x ] Yes
- [ ] No

**Pervious version**
3.3.5

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [x] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [x ] HOOBS Box
- [ ] HOOBS SD Card
- [ ] Custom Install

**Post your config**
    "accessories": [],
    "platforms": [
        {
            "platform": "Hue",
            "name": "Hue",
            "brightnessAdjustment": 100,
            "heartrate": 5,
            "lowBattery": 25,
            "timeout": 5,
            "waitTimePut": 50,
            "waitTimePutGroup": 1000,
            "waitTimeResend": 300,
            "waitTimeUpdate": 20,
            "anyOn": false,
            "configuredName": false,
            "lights": true,
            "users": {
                "001788FFFE24F66": "nfLeKB9H3rbyYDtrhBPsfx6CZJxrBfk353eOKN9"
            },
            "group0": false,
            "scenes": false,
            "scenesAsSwitch": false
        }
    ].

```
POST CONFIG HERE
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
POST LOG HERE
7/19/2021, 6:21:25 PMHue Hoobs BridgeLoaded plugin 'homebridge-hue'
7/19/2021, 6:21:25 PMHue Hoobs BridgeLoading 1 platforms...
7/19/2021, 6:21:25 PMHue Hoobs BridgeHuehomebridge-hue v0.13.18, node v14.17.2, homebridge v1.3.4
7/19/2021, 6:21:25 PMHue Hoobs BridgeHuesearching bridges and gateways
7/19/2021, 6:21:30 PMHue Hoobs BridgeHueVision: Signify Netherlands B.V. BSB002 bridge v1945163030, api v1.45.0
7/19/2021, 6:21:31 PMHue Hoobs BridgeHueVision: 1 accessories
7/19/2021, 6:21:31 PMHue Hoobs BridgeHuemasked debug info dumped to /var/lib/hoobs/huehoobsbridge/homebridge-hue.json.gz
7/19/2021, 6:21:31 PMHue Hoobs BridgeHueVision: 2 services
7/19/2021, 6:21:31 PMHue Hoobs BridgeBridge is running on port 55826.
7/19/2021, 6:22:48 PMHue Hoobs BridgeConfiguration change
7/19/2021, 6:22:50 PMHue Hoobs BridgeHuegoodbye
7/19/2021, 6:23:00 PMHue Hoobs BridgeLoaded plugin 'homebridge-hue'
7/19/2021, 6:23:00 PMHue Hoobs BridgeLoading 1 platforms...
7/19/2021, 6:23:00 PMHue Hoobs BridgeHuehomebridge-hue v0.13.18, node v14.17.2, homebridge v1.3.4
7/19/2021, 6:23:00 PMHue Hoobs BridgeHuesearching bridges and gateways
7/19/2021, 6:23:05 PMHue Hoobs BridgeHueVision: Signify Netherlands B.V. BSB002 bridge v1945163030, api v1.45.0
7/19/2021, 6:23:05 PMHue Hoobs BridgeHueVision: 1 accessories
7/19/2021, 6:23:05 PMHue Hoobs BridgeHuemasked debug info dumped to /var/lib/hoobs/huehoobsbridge/homebridge-hue.json.gz
7/19/2021, 6:23:05 PMHue Hoobs BridgeHueVision: 2 services
7/19/2021, 6:23:05 PMHue Hoobs BridgeBridge is running on port 55826.
7/19/2021, 6:38:56 PMHue Hoobs BridgeConfiguration change
7/19/2021, 6:38:58 PMHue Hoobs BridgeHuegoodbye
7/19/2021, 6:39:08 PMHue Hoobs BridgeLoaded plugin 'homebridge-hue'
7/19/2021, 6:39:08 PMHue Hoobs BridgeLoading 1 platforms...
7/19/2021, 6:39:08 PMHue Hoobs BridgeHuehomebridge-hue v0.13.18, node v14.17.2, homebridge v1.3.4
7/19/2021, 6:39:08 PMHue Hoobs BridgeHuesearching bridges and gateways
7/19/2021, 6:39:13 PMHue Hoobs BridgeHueVision: Signify Netherlands B.V. BSB002 bridge v1945163030, api v1.45.0
7/19/2021, 6:39:13 PMHue Hoobs BridgeHueVision: 1 accessories
7/19/2021, 6:39:13 PMHue Hoobs BridgeHuemasked debug info dumped to /var/lib/hoobs/huehoobsbridge/homebridge-hue.json.gz
7/19/2021, 6:39:13 PMHue Hoobs BridgeHueVision: 2 services
7/19/2021, 6:39:13 PMHue Hoobs BridgeBridge is running on port 55826.
7/19/2021, 6:39:14 PMHue Hoobs BridgeHueWARNINGheartbeat 0, drift 330
7/19/2021, 6:39:15 PMHue Hoobs BridgeHueWARNINGheartbeat 1, drift 267
**Additional context**
Add any other context about the problem here.

**Comment by DaDD65** [comment]
Had this issue too.  Was advised to check expose lights in the configuration page of Hue

**Comment by cdaniels77** [comment]
I have expose lights checked under Bridges/Gateways and no luck. only thing showing up is my hue bridge

**Comment by cdaniels77** [comment]
Also in homekit it i see the Hue Bridge though it says not supported

**Comment by DaDD65** [comment]
This is what I did.  I removed the Hue bridge from Hoobs and started over re- adding it.  I'll assume you pressed the pair on the Hue bridge and followed the config setup.  Once done I went into the config and Checked Expose lights, Expose all lights and Expose Groups as I had grouped a number my Hue bulbs grouped in fixtures.  Also I paired Hue with the Home app using the QR code on the Hue bridge page.  Restarted the bridge.  My lights appeared after that.  Might of restarted Hoobs too.  The problem after that was I had duplicates of the lights in Home as they were already there because Home supports Hue natively.  Fixed that by removing the Hue bridge from Home.  Duplicates were removed too.
